client/components/Nav/Nav.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import './Nav.css';
+
+const Nav = () => {
+  return (
+    <nav className="nav">
+      <h3 className="nav__script">Script</h3>
+      <h3 className="nav__diagram">ER Diagram</h3>
```

Comment:
Going `id` instead of `className`, imo may be better. Down the road if we want to add tests, finding an element by ID is much easier than class names.

Reply:
Also, do we have a strategy for switching between the script and ER diagram? It's fine that you don't have this implemented right now but I just want to make sure we have a strategy

Reply:
Yes :). The strategy is simple. Use a useState hook to determine when to add a special class "active". Frontend will create a state var called activeTabScript and declare it to false. Then in the JSX when this variable is false, we will conditionally render a class named "active" which just has the same border bottom that appears when the elements are hovered. Then we need to set up two click handlers, one for script and another for diagram. When script is clicked, activeTabScript is set to true. When diagram is clicked, it is set to false.